packages/core/src/domain/configuration/endpointBuilder.ts

```diff
@@ -26,6 +26,13 @@ const INTAKE_TRACKS = {
 
 export type EndpointType = keyof typeof ENDPOINTS[IntakeType]
 
+const INTAKE_DATACENTERS: { [site: string]: string } = {
+  'datadoghq.eu': 'eu1.prod.dog',
+  'datadoghq.com': 'us1.prod.dog',
+  'us3.datadoghq.com': 'us3.prod.dog',
+  'us5.datadoghq.com': 'us5.prod.dog',
```

Comment:
I would be in favor of a `replica.datacenter` Init configuration, so we don't need to maintain a list here. Usage example:

```ts
DD_RUM.init({
  ....
  site: 'datadoghq.eu',
  replica: {
    ....
    datacenter: 'eu1.prod.dog'
  }
})
```

Reply:
Good point!